Packs/FeedIntel471/README.md

```diff
@@ -1 +1,5 @@
-Note: Support for this pack will be moving to the partner around 2021-04-23.
+Note: Support for this pack moved to the partner on 2021-05-05.  Please contact the partner directly via the support link on the right.
+
+# Intel 471
```

Comment:
Please dont use H1 here @rhallick, H4 or 5 is best. 
And plz see the template I slacked you.